### ISSUE TITLE
dependency issue when running 1.3.0 on github actions

### ISSUE THREAD

**Issue by rogervdf** [JoelLefkowitz/random-env#1]
```
node:internal/modules/cjs/loader:1031
  throw err;
  ^

Error: Cannot find module '@actions/core'
Require stack:
- /home/runner/work/_actions/joellefkowitz/random-env/v1.3.0/publish/main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1028:[15](https://github.com/reconfigure-ai/asset-monitor/actions/runs/9518263851/job/26238769906#step:9:16))
    at Function.Module._load (node:internal/modules/cjs/loader:873:27)
    at Module.require (node:internal/modules/cjs/loader:1100:19)
    at require (node:internal/modules/cjs/helpers:119:[18](https://github.com/reconfigure-ai/asset-monitor/actions/runs/9518263851/job/26238769906#step:9:19))
    at Object.<anonymous> (/home/runner/work/_actions/joellefkowitz/random-env/v1.3.0/publish/main.js:3:12)
    at Module._compile (node:internal/modules/cjs/loader:1[19](https://github.com/reconfigure-ai/asset-monitor/actions/runs/9518263851/job/26238769906#step:9:20)8:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/joellefkowitz/random-env/v1.3.0/publish/main.js'
  ]
}
```

**Comment by JoelLefkowitz** [comment]
Thanks for raising @rogervdf I'll take a look

**Comment by JoelLefkowitz** [comment]
Fixed now in v1.4.2
